HISTORY.md

```diff
@@ -1,3 +1,12 @@
+unreleased
+==================
```

Comment:
The formatting is messed up here: just shorten the line and add a blank line after this and before the bullet point.